src/interpreter_funcs/voxel_noise.rs

```diff
@@ -78,9 +78,9 @@ impl Func for FuncVoxelNoise {
                 refinement: ParamRefinement::Float3(Float3ParamRefinement {
                     min_value: None,
                     max_value: None,
-                    default_value_x: Some(1.0),
-                    default_value_y: Some(1.0),
-                    default_value_z: Some(1.0),
+                    default_value_x: Some(10.0),
+                    default_value_y: Some(10.0),
+                    default_value_z: Some(20.0),
```

Comment:
Are these block start and block end changes on purpose?

Reply:
because the shadow is ugly when the geometry intersects the ground plane. therefore I put it above

Reply:
If you want to, we can be inconsistent here and keep it like this, but a lot of other things are not snapped to ground by default (and therefore ugly). What do we want to do about that?

Reply:
The other generators create primitives and it makes sense to create unitized versions of them. I can't imagine a unit version of a noise. I want the user to get an instantly gratifying result. Sure, we can wrap it around zero, but once again, what would it be good for in this case? Moreover, it is 10 times larger than any other geometry generated inside of our software (there is no good solution to this - either the noise is big or the voxels are too small, or the result looks like a single voxel).

That brings me to another issue - we know the models that will be most probably imported int our software and all of them are thousands of units big. Shouldn't we set our defaults to generate roughly 1000 units large results and operate correctly on such inputs?

Reply:
> Shouldn't we set our defaults to generate roughly 1000 units large results and operate correctly on such inputs?

That's what I was aiming at.

But let's do it in another task and another PR to prevent this one from getting intractable.
https://trello.com/c/VtG2C1Le/333-adjust-default-values-in-operations-for-more-instantly-gratifying-results

Reply:
Also, let's keep this one as is. We can potentially adjust it in the other task, if needed.